src/styles/blocks/calendar.scss

```
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  align-content: flex-start;
+  flex-wrap: wrap;
+  gap: $indentation-between-days;
+
+  width: $day-size * 7 + $indentation-between-days * 6 + $calendar-paddings * 2;
+  height: $day-size * 5 + $indentation-between-days * 4 + $calendar-paddings * 2;
+
+  background-color: #fff;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: $day-size;
+    width: $day-size;
+
+    border: solid $day-border-size $day-border-color;
+    background-color: $day-background-color;
+    transition: all $transition-duration ease-in-out;
```

don't use it like that, because it's the same as transition: 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```

---

try to change `calendar--month-length--31` to 30 or 29 in devTools, you will see that animation would be with duration